pyxrf/model/load_data_from_db.py

```diff
@@ -432,7 +514,9 @@ def map_data2D_srx(runid, fpath,
             data1 = db.get_table(hdr, fill=True, stream_name='primary')
             if num_end_lines_excluded is not None:
                 data1 = data1[:datashape[0]]
-            if ypos_name not in data1.keys() and 'E_tomo' not in start_doc['scaninfo']['type']:
+            # if ypos_name not in data1.keys() and 'E_tomo' not in start_doc['scaninfo']['type']:
+            #print(f"data1 keys: {data1.keys()}")
```

Comment:
Can those debug prints be removed?
```suggestion
```

Reply:
This is work in progress, so I would like to keep the debug prints for now. This code will have to be rewritten in the nearest future anyway and then this functionality will be moved from PyXRF to respective suitcase(s).